modules/ROOT/pages/configuration/server/occ_app_command.adoc

```diff
@@ -421,56 +420,56 @@ All of the available keys, along with default values for configValue, are listed
 
 [width="70%",cols=",",options="header",]
```

Comment:
Interesting. I wonder why GitHub renders some of the table italicised.

Reply:
No clue, I double checked but I can´t find the reason.

Reply:
Well, that's what counts.